src/index.js

```diff
@@ -17,6 +17,14 @@ export default function configureStore (middlewares = []) {
           return actions
         },
 
+        getAction (type) {
```

Comment:
I would suggest to name the method as `getActionByType` and maybe to add method `getAction` that accepts the function as a predicate, so the usage would be like: `getActions(action => action.type === myType)`
That would give much more flexibility to developers and will allow them to implement their own comparison functions.
What do you think?

Reply:
I like the recommendation and I'll do the change soon.

Reply:
Done, have a look.